cmd/notation/internal/display/metadata/json/inspect.go

```diff
@@ -0,0 +1,204 @@
+// Copyright The Notary Project Authors.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package json
+
+import (
+	"crypto/sha256"
+	"crypto/x509"
+	"encoding/hex"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/notaryproject/notation-core-go/signature"
+	"github.com/notaryproject/notation-go/plugin/proto"
+	"github.com/notaryproject/notation/cmd/notation/internal/output"
+	"github.com/notaryproject/notation/internal/envelope"
+	"github.com/notaryproject/tspclient-go"
+
+	ocispec "github.com/opencontainers/image-spec/specs-go/v1"
+)
+
+type inspectOutput struct {
+	MediaType  string `json:"mediaType"`
+	Signatures []Signature
+}
+
+// Signature is the signature envelope for printing in human readable format.
+type Signature struct {
+	MediaType             string             `json:"mediaType"`
+	Digest                string             `json:"digest,omitempty"`
+	SignatureAlgorithm    string             `json:"signatureAlgorithm"`
+	SignedAttributes      map[string]any     `json:"signedAttributes"`
+	UserDefinedAttributes map[string]string  `json:"userDefinedAttributes"`
+	UnsignedAttributes    map[string]any     `json:"unsignedAttributes"`
+	Certificates          []Certificate      `json:"certificates"`
```

Comment:
Similarly, it can be
```suggestion
	Certificates          []*Certificate      `json:"certificates"`
```